### ISSUE TITLE
Implement updates page filtering

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#705]
```
Use https://code.google.com/p/mdv/source/browse/#svn%2Ftrunk%2Fcapabilities
```

Original issue reported on code.google.com by `ericbide...@html5rocks.com` on 2 Nov 2011 at 4:02

**Comment by GoogleCodeExporter** [comment]
```
Revision:
http://code.google.com/p/html5rocks/source/detail?r=83c6713de63296b3bec8718eb152
68de38aaf6ea

adds resource type formatting. Tag filtering via input is already there.
```

Original comment by `ericbide...@html5rocks.com` on 14 Nov 2011 at 2:38
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 1 Dec 2011 at 12:45
- Removed labels: **Updates**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 18 Jan 2012 at 11:43

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 23 Jan 2012 at 6:12
- Added labels: **Priority-P1**
- Removed labels: **Priority-P2**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in revision 0746aa5958b5 plus revision 16646aef6dc6.
```

Original comment by `han...@google.com` on 25 Jan 2012 at 4:38
- Changed state: **Fixed**
